rfcs/text/0001-wildcard-data-type.md

```diff
@@ -21,29 +28,10 @@ For a field to use wildcard, it will require changing the field's defined schema
 
 | Field Set | Field(s) |
 | --------- | -------- |
-| [`agent`](0001/agent.yml) | `agent.build.original` |
-| [`as`](0001/as.yml) | `as.organization.name` |
-| [`client`](0001/client.yml) | `client.domain`<br> `client.registered_domain` |
-| [`destination`](0001/destination.yml) | `destination.domain`<br> `destination.registered_domain` |
-| [`dns`](0001/dns.yml) | `dns.question.name`<br> `dns.answers.data` |
-| [`error`](0001/error.yml) | `error.stack_trace`<br> `error.type` |
-| [`file`](0001/file.yml) | `file.directory`<br> `file.path`<br> `file.target_path` |
-| [`geo`](0001/geo.yml) | `geo.name` |
-| [`host`](0001/host.yml) | `host.hostname`<br> |
-| [`http`](0001/http.yml) | `http.request.referrer`<br> `http.request.body.content`<br> `http.response.body.content` |
-| [`log`](0001/log.yml) | `log.file.path`<br> `log.logger` |
-| [`organization`](0001/organization.yml) | `organization.name` |
-| [`os`](0001/os.yml) | `os.name`<br> `os.full` |
-| [`pe`](0001/pe.yml) | `pe.original_file_name` |
-| [`process`](0001/process.yml) | `process.command_line`<br> `process.executable`<br> `process.name`<br> `process.thread.name`<br> `process.title`<br> `process.working_directory`<br> |
-| [`registry`](0001/registry.yml) | `registry.key`<br> `registry.path`<br> `registry.data.strings` |
```

Comment:
This change makes sense when I think about it, because these wildcards are often not leading wildcards, so they are less costly to do on a keyword fields.

Reply:
`registry.data.strings` though, could be worth keeping

Reply:
Ok, I'm good with keeping `registry.data.strings`. I'll re-add.